src/Http/Controllers/ResetController.php

```diff
@@ -2,21 +2,35 @@
 
 namespace Whitecube\LaravelCookieConsent\Http\Controllers;
 
-use Whitecube\LaravelCookieConsent\CookiesManager;
+use Illuminate\Support\Facades\Cookie;
 use Illuminate\Http\Request;
+use Whitecube\LaravelCookieConsent\CookiesManager;
+use Whitecube\LaravelCookieConsent\Facades\Cookies;
```

Comment:
We do not need the `Cookies` facade since the `CookieManager` is injected in the controller's `__invoke` method.